fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadJob.java

```diff
@@ -634,12 +637,14 @@ protected void logFinalOperation() {
                         state, failMsg));
     }
 
-    public void unprotectReadEndOperation(LoadJobFinalOperation loadJobFinalOperation) {
+    public void unprotectReadEndOperation(LoadJobFinalOperation loadJobFinalOperation, boolean replay) {
+        if (replay) {
```

Comment:
Where to use the state ?

Reply:
If current state is COMMITTED, and it is going to become FINISHED. 
It will call replayTxnAttachment.
![image](https://user-images.githubusercontent.com/31178013/188532257-844b2a41-bc4d-4025-a577-8022600d3b01.png)
which will set the state to be LOADING(because last end operation's state is LOADING)